src/components/EditTab/index.js

```diff
@@ -90,7 +93,10 @@ export default class EditTab extends React.PureComponent {
           formInstance,
           permissions: {
             crudOperations: permissions
-          }
+          },
+          standardOpsConfig: {
+            delete: deleteConfig = {}
+          } = {}
```

Comment:
**standardOpsConfig** is always defined and does not need default value

Reply:
Nope, it's undefined on Create or Show view.